### ISSUE TITLE
Couldn't find models gradients values !

### ISSUE THREAD

**Issue by AhmedAl93** [amzn/convolutional-handwriting-gan#19]
Hello,
Thanks for your amazing work !

When training on my custom dataset, I wanted to plot the gradients of each model's loss function using tensorboard, but I was confused when I found out that for "_parameters" field is empty for netG, netD & netOCR models, same goes for "_state_dict_hooks" field.
Here are some log lines:

`End of epoch 3 / 200 Time Taken: 171 sec
learning rate = 0.0002000
Model netOCR _parameters: {}
Model netOCR _state_dict_hooks: OrderedDict()

Model netG _parameters: OrderedDict()
Model netG G_param: SN
Model netG _state_dict_hooks: OrderedDict()

Model netD _parameters: OrderedDict()
Model netD D_param: SN
Model netD _state_dict_hooks: OrderedDict()`

Can anyone explain this, please?

**Comment by rlit** [comment]
not sure I see why this would happen, but this implementation writes on default into a 'visdom' server, which can replace tensorboard in most cases - did you try to use that?
